src/WCSimConstructCylinder.cc

```diff
@@ -825,19 +840,16 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinder()
 						false,0,
 						checkOverlaps);
 
-	//G4LogicalSkinSurface *WaterTyTVSurfaceBot =
-	new G4LogicalSkinSurface("WaterTyTVSurfaceBot", logicWCTVTyvek, OpWaterTySurface);
-
-	//Top
-	//G4VPhysicalVolume* physiWCTVTyvekTop =
-	new G4PVPlacement(	0,
-						G4ThreeVector(0.,0.,0.5*m
-									  +WCTyvekThickness/2),
-						logicWCTVTyvek,
-						"WCTVTyvekTop",
-						logicWCTopVeto,
-						false,0,
-						checkOverlaps);
+          G4LogicalSkinSurface *WaterTyTVSurfaceBot = new G4LogicalSkinSurface("WaterTyTVSurfaceBot", logicWCTVTyvek, OpWaterTySurfaceInWallBottom);
+	  //Top
+	  G4VPhysicalVolume* physiWCTVTyvekTop =
+			new G4PVPlacement(	0,
+		                  		G4ThreeVector(0.,0.,0.5*m
+													+WCTyvekThickness/2),
+								logicWCTVTyvek,
+		               			"WCTVTyvekTop",
+		          				logicWCTopVeto,
+				 				false,0,true);
```

Comment:
```suggestion
				 				false,0,checkOverlaps);
```
We also don't want to check for overlaps every time we run - only when we want to